docs/user/openstack/kuryr.md

```diff
@@ -76,4 +81,4 @@ There are known limitations when using Kuryr SDN:
 * There is an amphora load balancer VM being deployed per OpenShift service with the default Octavia load balancer driver (amphora driver). If the environment is resource constrained it could be a problem to create a large amount of services.
 * Depending on the Octavia version, UDP listeners are not supported. This means that OpenShift UDP services are not supported.
 * There is a known limitation of Octavia not supporting listeners on different protocols (e.g., UDP and TCP) on the same port. Thus services exposing the same port for different protocols are not supported.
-* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP for DNS resolution (`use-vc` option at `resolv.conf`). This may be a problem for pods running Go applications compiled with `CGO_DEBUG` flag disabled as that forces to use the `go` resolver that is only using UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a problem also for musl-based containers as it's resolver does not support `use-vc` option. This would include e.g., images build from `alpine`.
+* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP for DNS resolution (`use-vc` option at `resolv.conf`). This may be a problem for pods running Go applications compiled with `CGO_ENABLED` flag disabled as that forces to use the `go` resolver that is only using UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a problem also for musl-based containers as it's resolver does not support `use-vc` option. This would include e.g., images build from `alpine`.
```

Comment:
Rather than `compiled with CGO_ENABLED flag disabled`, I find `compiled without the CGO_ENABLED flag` easier to parse.

Reply:
Problem is when compiling it with it disable (i.e., CGO_ENABLED=0), if you compile it without the flag, or with the flag enabled (CGO_ENABLED=1) it should work